core/templates/dev/head/pages/exploration_editor/exploration_editor.html

```diff
@@ -324,6 +326,7 @@
   <script src="{{TEMPLATE_DIR_PREFIX}}/domain/exploration/ParamSpecObjectFactory.js"></script>
   <script src="{{TEMPLATE_DIR_PREFIX}}/domain/exploration/ParamSpecsObjectFactory.js"></script>
   <script src="{{TEMPLATE_DIR_PREFIX}}/domain/exploration/ParamTypeObjectFactory.js"></script>
+  <script src="{{TEMPLATE_DIR_PREFIX}}/domain/exploration/AnswerStatsFactory.js"></script>
```

Comment:
AnswerStatsObjectFactory.js and put it alphabetically

Reply:
Done.